app/lib/dfe/reference_data/degrees.rb

```diff
@@ -1,730 +1,669 @@
+          '551c390b-d378-4d41-ad0b-4caf800804d2' => { 'name' => 'O level - grade A, B or C', synonyms => [], level => '2' },
+          '7811d3e0-c1c2-4fb8-9d23-844ceab0764a' => { 'name' => 'A level', synonyms => [], level => '3' },
+          '39b9f3c2-00cc-4408-85c8-1972726c728f' => { 'name' => 'access to higher education diploma', synonyms => [], level => '3' },
+          'c4282161-2358-429c-baba-b7765b02262c' => { 'name' => 'advanced apprenticeship', synonyms => [], level => '3' },
+          '14c9c9f8-3b44-4e3d-bf4c-17cded6a98fe' => { 'name' => 'applied general', synonyms => [], level => '3' },
+          '4646546e-5d85-47b6-93db-b844a3cec56c' => { 'name' => 'AS level', synonyms => [], level => '3' },
+          '414ac908-1e70-4b13-834f-936cbdb995cc' => { 'name' => 'international Baccalaureate diploma', synonyms => [], level => '3' },
+          '5ace8fac-63be-457c-918d-f048e23fa2e0' => { 'name' => 'level 3 award', synonyms => [], level => '3' },
+          'fa3dbbe7-8ea6-4787-83b9-a8118672666d' => { 'name' => 'level 3 certificate', synonyms => [], level => '3' },
+          '98db840e-a399-40c5-a95d-589e54ccee4c' => { 'name' => 'level 3 diploma', synonyms => [], level => '3' },
+          '0ab5bb24-3cde-4489-96ff-fd558c046474' => { 'name' => 'level 3 English for speakers of other languages', synonyms => ['ESOL'], level => '3' },
+          '537433b3-562e-4cac-b61e-b9386a80ace0' => { 'name' => 'level 3 national certificate', synonyms => [], level => '3' },
+          '67bc4f81-186a-4b50-a09d-4b50d64c4c0a' => { 'name' => 'level 3 national diploma', synonyms => [], level => '3' },
+          '1b2063df-13b3-4e46-8020-916708493273' => { 'name' => 'level 3 national vocational qualification', synonyms => ['NVQ'], level => '3' },
+          '1eb92c31-cd60-4ff6-8d0f-8dfde9705191' => { 'name' => 'music grades 6, 7 and 8', synonyms => [], level => '3' },
+          'a42468af-a30e-4893-852e-fa8aad0047a5' => { 'name' => 'tech level', synonyms => [], level => '3' },
+          'f7ab37d2-4413-41e6-b0aa-9b0cb7bf9d53' => { 'name' => 'certificate of higher education', synonyms => ['CertHE'], level => '4' },
+          'dbe7bb34-2df5-413a-8ce9-9b806da829e1' => { 'name' => 'higher apprenticeship', synonyms => [], level => '4' },
+          '8f08ac5c-5657-4616-ae97-146b5e6f5b67' => { 'name' => 'higher national certificate', synonyms => ['HNC'], level => '4' },
+          'aca93aaf-747b-489e-a76f-f25310e6f87d' => { 'name' => 'level 4 award', synonyms => [], level => '4' },
+          'de7c4d54-68bd-44c3-a8fe-70a9ff1847a4' => { 'name' => 'level 4 certificate', synonyms => [], level => '4' },
+          '1eb3625e-d376-4b0b-8521-8a1269ef73ac' => { 'name' => 'level 4 diploma', synonyms => [], level => '4' },
+          'b6ba9cfe-5336-4589-a466-674dded38563' => { 'name' => 'level 4 national vocational qualification', synonyms => ['NVQ'], level => '4' },
```

Comment:
I'd suggest we drop everything below level 5, as those qualifications are never called a degree or considered equivalent to a degree? Either that or we should move this list out of the `ReferenceData::Degree` namespace...

Reply:
I've moved it to `ReferenceData::Qualifications`